syft/frameworks/torch/hook/hook.py

```diff
@@ -529,6 +523,7 @@ def _transfer_methods_to_native_tensor(cls, tensor_type: type, syft_type: type):
             "__ge__",
             "__lt__",
             "__le__",
+            "data",
```

Comment:
I don't think you have to do this, you're already directly redefining `torch.nn.Parameter.data`